docs: fix wording in eager loading section [ci skip] (#44188)
rails/rails: guides/source/active_record_querying.md

end
```

> The above code will execute just **2** queries, as opposed to the **11** queries from the original case:

Reviewer: We should also change line 1412 in the `includes` section to match this one.

Author: Ah, thanks for catching this one!

Author: Fixed.

```sql

> SELECT `books`.* FROM `books` LIMIT 10

Reviewer: We should fix this same mistake on line 1415 in the `includes` section as well.

Author: Good catch.

Author: Fixed.

SELECT `authors`.* FROM `authors`
  WHERE `authors`.`book_id` IN (1,2,3,4,5,6,7,8,9,10)
```

NOTE: The `preload` method uses an array, hash, or a nested hash of array/hash in the same way as the `includes` method to load any number of associations with a single `Model.find` call. However, unlike the `includes` method, it is not possible to specify conditions for preloaded associations.

### eager_load

> With `eager_load`, Active Record ensures associations are loaded using　`LEFT OUTER JOIN` for all specified associations.

Reviewer: ```suggestion
With `eager_load`, Active Record loads all specified associations using a `LEFT OUTER JOIN`.
```

  WHERE `books`.`id` IN (1,2,3,4,5,6,7,8,9,10)
```

> NOTE: The `eager_load` method uses an array, hash, or a nested hash of array/hash in the same way as the `includes` method to load any number of associations with a single `Model.find` call. Also, like the `includes` method, you can specify the conditions of the eager loaded association.

Reviewer: ```suggestion
NOTE: The `eager_load` method uses an array, hash, or a nested hash of array/hash in the same way as the `includes` method to load any number of associations with a single `Model.find` call. Also, like the `includes` method, you can specify conditions for eager loaded associations.
```
